cpp/serve/config.cc

```diff
@@ -0,0 +1,121 @@
+/*!
+ *  Copyright (c) 2023 by Contributors
+ * \file serve/config.cc
+ */
+#define PICOJSON_USE_INT64
+
+#include "config.h"
+
+#include <picojson.h>
+
+#include "data.h"
+
+namespace mlc {
+namespace llm {
+namespace serve {
+
+/****************** GenerationConfig ******************/
+
+TVM_REGISTER_OBJECT_TYPE(GenerationConfigNode);
+
+GenerationConfig::GenerationConfig(String config_json_str) {
```

Comment:
Although we don't need to support all of these at the first cut, I think it would be nice to define and document for the next iteration. 
* `stream`: Whether to stream back partial progress.
* `n`: How many completions to generate for each prompt.
* `user` (optional): A unique identifier representing your end-user, which can help monitoring.

Reply:
Thanks for the suggestions. Among them, `stream` will be handled in the callback function in Request.